x-pack/packages/security-solution/features/src/security/kibana_sub_features.ts

```diff
@@ -660,20 +661,47 @@ export const getSecurityBaseKibanaSubFeatureIds = (
 export const getSecuritySubFeaturesMap = ({
   experimentalFeatures,
 }: SecurityFeatureParams): Map<SecuritySubFeatureId, SubFeatureConfig> => {
+  const enableSpaceAwarenessIfNeeded = (subFeature: SubFeatureConfig): SubFeatureConfig => {
+    if (experimentalFeatures.endpointManagementSpaceAwarenessEnabled) {
+      subFeature.requireAllSpaces = false;
+      subFeature.privilegesTooltip = undefined;
+    }
+
+    return subFeature;
+  };
+
   const securitySubFeaturesList: Array<[SecuritySubFeatureId, SubFeatureConfig]> = [
```

Comment:
I wonder if it wouldn't be more readable if you iterate over `securitySubFeaturesList` instead of wrapping each subFeature in `enableSpaceAwarenessIfNeeded`. What do you think?

Reply:
I guess. It accomplishes the same thing, but I'm open to changing it. 

what do you find about this implementation that makes it less readable?

I did try to make the function names clear as to what they do and also opted to do it this way only because it gives us some flexibility if we ever have a case where a specific feature privilege does require it to be space agonistic.

Reply:
I was thinking more about doing it in one place insterad of passing to all separately:

```
  const securitySubFeaturesListWithSpaceAwarnessIfNeeded = securitySubFeaturesList.map(
    ([id, config]) => [id, enableSpaceAwarenessIfNeeded(config)];
   );
  ```

Reply:
yeah... Not sure if its better - I don't like that additional iteration by using `.map()` and recreating the entire structure again. 

I'm going to leave it as is for now.